components/atom/button/src/index.js

```diff
@@ -45,19 +56,38 @@ const AtomButton = (props) => {
     className
   )
   const newProps = cleanProps(props)
-  return (<button {...newProps} className={classNames} disabled={disabled}>
-    {leftIcon && <span className={`${CLASS}-leftIcon`}>{leftIcon}</span>}
-    {leftIcon || rightIcon
-      ? <span className={`${CLASS}-text`}>{children}</span>
-      : children
-    }
-    {rightIcon && <span className={`${CLASS}-rightIcon`}>{rightIcon}</span>}
-  </button>)
+
+  const Button = ({ children, url, disabled, ...attrs }) => link
+    ? <Link {...attrs} href={url}>{children}</Link>
+    : <button {...attrs} disabled={disabled}>{children}</button>
+
+  return (
+    <Button {...newProps} className={classNames} title={title} disabled={disabled}>
+      {leftIcon && <span className={`${CLASS}-leftIcon`}>{leftIcon}</span>}
+      {leftIcon || rightIcon
+        ? <span className={`${CLASS}-text`}>{children}</span>
+        : children
+      }
+      {rightIcon && <span className={`${CLASS}-rightIcon`}>{rightIcon}</span>}
+    </Button>
+  )
 }
 
 AtomButton.displayName = 'AtomButton'
 
 AtomButton.propTypes = {
+  /**
+   * HTML element: if true, render a link. Otherwise render a button.
+   */
+  link: PropTypes.bool,
```

Comment:
I would say that if the `url` string is passed, the `link` prop is implicit. Isn't?